git-tar/function/handler.go

```diff
@@ -24,29 +24,40 @@ func Handle(req []byte) []byte {
 		os.Exit(-1)
 	}
 
+	statusEvent := sdk.BuildEventFromPushEvent(pushEvent)
+	status := sdk.BuildStatus(statusEvent, "")
```

Comment:
I don't know what `""` signifies from reading the code. Can you explain please?

Reply:
The function definition is
```go
func BuildStatus(event *Event, token string) *Status
``` 
It allows to pass and reuse an auth token if available. Currently, we are **not passing** `authtoken` between consecutive functions although the SDK allows to do it.